db/migrate/20160126151325_add_disk_and_memory_minimum_to_hardware.rb

```diff
@@ -0,0 +1,6 @@
+class AddDiskAndMemoryMinimumToHardware < ActiveRecord::Migration
+  def change
+    add_column :hardwares, :disk_size_minimum, :integer, :limit => 8
```

Comment:
maybe :disk_size_gb_minimum ? Though seems like other attributes on Hardware model don't have unit in it, so not sure what is better :-)